src/renderer/init.js

```diff
@@ -29,6 +29,7 @@ import dbMiddleware from "~/renderer/middlewares/db";
 import createStore from "~/renderer/createStore";
 import events from "~/renderer/events";
 import { setAccounts } from "~/renderer/actions/accounts";
+import { loadLSS } from "~/renderer/modals/FullNode/helpers";
```

Comment:
I don't like the fact this loadLSS is in a hidden place like modals/FullNode/helpers. I would prefer we put it next to the other loaders, probably in renderer/storage.js itself.

Reply:
Moved them